### PR TITLE
Fut/compare

### PR DESCRIPTION
**Pull request by Delfil**
An application to automatically test and compare networks and/or datasets.

### CODE REVIEW

File: src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+public class EvaluationFileUtil {
+	protected static final DecimalFormat SEED_FORMATER = new DecimalFormat(
+			"S+0000000000000000000;S-0000000000000000000");
+	protected static final DecimalFormat EPOCH_FORMATER = new DecimalFormat("E000");
+
+	private static final String F = "/";
+	private static final Object EVAL_EXTENTION = ".eval";
+
+	public static String getEvalFileName(long seed, int epoch) {
```
Comment (paulbakker65) [inline src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java]: Javadoc missing

File: src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		return new StringBuilder(SEED_FORMATER.format(seed)).append(EPOCH_FORMATER.format(epoch)).append(EVAL_EXTENTION)
+				.toString();
+	}
+
+	public static String getEvalPathName(Data data, FNNCBuilder builder) {
+		return new StringBuilder(builder.getPathName()).append(F).append(data.getTimeStamp()).append(F).toString();
+	}
+
+	public static Evaluation<Double> load(long seed, int epoch, Data data, FNNCBuilder builder) {
```
Comment (paulbakker65) [inline src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java]: Javadoc missing

File: src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+			} catch (IOException | ClassNotFoundException e) {
+				e.printStackTrace();
+			}
+			return null;
+		} else {
+			return null;
+		}
+	}
+
+	public static List<Evaluation<Double>> load(int epoch, Data data, FNNCBuilder builder) {
```
Comment (paulbakker65) [inline src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java]: Javadoc missing

File: src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+			e.printStackTrace();
+		}
+		return results;
+	}
+
+	public static boolean evalExistst(long seed, int epoch, Data data, FNNCBuilder builder) {
+		return new File(getEvalPathName(data, builder) + getEvalFileName(seed, epoch)).exists();
+	}
+
+	public static void save(Evaluation<Double> eval, long seed, int epoch, Data data, FNNCBuilder builder) {
```
Comment (paulbakker65) [inline src/main/java/nl/tudelft/bep/deeplearning/network/result/EvaluationFileUtil.java]: Javadoc missing

**Comment by bottervanger** [comment]
prima

**Pull request merged**